packages/real/crates/real-base/src/file.rs

```diff
@@ -0,0 +1,16 @@
+pub mod native {
```

Comment:
Shouldn't this be part of `./real.rs`?

Reply:
Maybe. But it's very specific behavior regarding file reading.